README.md

```diff
@@ -79,6 +79,18 @@ self.addEventListener('periodicsync', event => {
 });
 ```
 
+## Removing a periodic sync when the user signs out
+```javascript
+// index.html
+
+navigator.serviceWorker.ready.then(registration => {
+  registration.periodicSync.getTags().then(tags => {
+    if (tags.includes('get-latest-news'))
+      registration.periodicSync.unregister('get-latest-news');
```

Comment:
Why wouldn't I just call unregister() w/o checking getTags() first?

Reply:
You could. I'm trying to demonstrate use of getTags() here.
I can separate out in two blocks.